src/BM2/DungeonBundle/Service/DungeonMaster.php

```diff
@@ -155,8 +155,8 @@ public function joinDungeon(Dungeoneer $dungeoneer, Dungeon $dungeon) {
 
 	public function startDungeon(Dungeon $dungeon) {
 		list($party, $missing, $wait) = $this->calculateTurnTime($dungeon);
-		if ($party<3) return; // don't start until we have at least 3 people
-		if ($missing > $party/2) return; // don't start until at least half the members have selected an action
+		if ($party<1) return; // party must consist of at least one dungeoneer to begin
```

Comment:
The game, if I understand it correctly, has a 0 member party exist in/at all dungeons, and I'd like to leave it so it can be used later for other things, like different dungeons having different minimum party sizes.